packages/labextension/src/widget.tsx

```diff
@@ -144,75 +146,74 @@ const KernelUsage = (props: {
       return !usage.hostname ? (
         <>
           <h3 className="jp-KernelUsage-section-separator">
-            Kernel usage details are not available
+            props.trans.__('Kernel usage details are not available')
```

Comment:
I think missing curly braces all over the board.

Reply:
:+1: